patches/0025-LPD-54093-Remove-IE11-fixes-causing-focus-issue-with.patch

```diff
@@ -0,0 +1,34 @@
+From 857f5383975ed670a36498db4eeda82b3b4b22ee Mon Sep 17 00:00:00 2001
+From: fortunato <fortunato.maldonado@liferay.com>
+Date: Fri, 2 May 2025 10:02:32 -0600
+Subject: [PATCH] LPD-54093 Remove IE11 fixes causing focus issue with
+ scrollbar
+
+---
+ plugins/wysiwygarea/plugin.js | 13 -------------
+ 1 file changed, 13 deletions(-)
+
+diff --git a/plugins/wysiwygarea/plugin.js b/plugins/wysiwygarea/plugin.js
+index 97c849a80c..45bc136008 100644
+--- a/plugins/wysiwygarea/plugin.js
++++ b/plugins/wysiwygarea/plugin.js
+@@ -268,19 +268,6 @@
+ 			removeSuperfluousElement( 'div' );
+ 		}
+ 
+-		// Fix problem with cursor not appearing in Webkit and IE11+ when clicking below the body (https://dev.ckeditor.com/ticket/10945, https://dev.ckeditor.com/ticket/10906).
+-		// Fix for older IEs (8-10 and QM) is placed inside selection.js.
+-		if ( CKEDITOR.env.webkit || ( CKEDITOR.env.ie && CKEDITOR.env.version > 10 ) ) {
```

Comment:
Issue title and the patch name confused me a bit, since IE11 fix should be behind environment `if` check, and not have any effect on code. The key bit here seems to be `CKEDITOR.env.webkit`, which applies the code to Safari and... Chrome on iOS and Linux? I thought Chrome does not use webkit, but apparently it does. My Chrome on Linux definitely does return true for `CKEDITOR.env.webkit`.

So, we can push this code as it is, but I'd make two small changes:
- Just remove `CKEDITOR.env.webkit` check, leaving fix for IE. Yeah, we don't care about IE, but why not leave in perfectly good fix when we already have it.
- I'd name patch something like `Do-not-apply-focus-fix-to-webkit`